### ISSUE TITLE
ImageMagick can't load PNG'S anymore

### ISSUE THREAD

**Issue by Febbe** [msys2/MINGW-packages#19715]
### Description / Steps to reproduce the issue

Following fails:
1. install  `pacman -Syuu mingw-w64-ucrt-x86_64-libpng`
2. install `pacman -Syuu mingw-w64-ucrt-x86_64-imagemagick`
3. `magick identify any-icon.png`
4. Or `magick convert any-icon.png any-icon.ico`

with:
```
identify: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/746.
...
convert: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/746.
convert: no images defined `any-icon.ico' @ error/convert./ConvertImageCommand/3362.
```

### Expected behavior

That ImageMagick will find the installed delegate.

### Actual behavior

Delegate is not found

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22621

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [X] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

**Comment by MehdiChinoune** [comment]
Fixed by #19707

**Comment by Biswa96** [comment]
Thank you for reporting the issue. The issue has been fixed recently in this commit https://github.com/msys2/MINGW-packages/commit/46ac18b5740f3c4622f8c48421f187aa229c8ec0

The new package is in queue stage https://packages.msys2.org/queue. Please wait for the new package.
